memstore/ingestion.go

```diff
@@ -295,7 +297,7 @@ func (shard *TableShard) insertPrimaryKeys(primaryKeyColumns []int, eventTimeCol
 				return nil, nil, nil, utils.StackError(err, "Failed to get column id for col %d", col)
 			}
 
-			for columnID >= len(shard.LiveStore.lastModifiedTimePerColumn) {
+			if columnID >= len(shard.LiveStore.lastModifiedTimePerColumn) {
```

Comment:
will this work  if columnID is not continuous..?

Reply:
good catch